homeassistant/components/zwave_js/entity.py

```diff
@@ -188,10 +188,19 @@ def generate_name(
         # Only include non empty additional info
         if additional_info := [item for item in (additional_info or []) if item]:
             name = f"{name} {' '.join(additional_info)}"
-        # append endpoint if > 1
-        if (
-            self.info.primary_value.endpoint is not None
-            and self.info.primary_value.endpoint > 1
+
+        # Only append endpoint to name if there are no equivalent values on a lower
```

Comment:
Isn't the logic the opposite?

```suggestion
        # Only append endpoint to name if there are equivalent values on a lower
```